models/brac/views/hhview_visits.sql

```diff
@@ -0,0 +1,27 @@
+SELECT reported, reported_by_parent, household_id FROM(
+
+SELECT 
+	upr.reported AS reported,
+	upr.reported_by_parent,
+	cvp.parent_uuid AS household_id
+FROM 
+	{{ ref("useview_patient_record") }} AS upr
+INNER JOIN {{ ref("contactview_person") }} cvp ON (upr.patient_id=cvp.patient_id)
+UNION ALL
```

Comment:
For readability,  let's have a newline preceding and coming after the 'union all' statements